### ISSUE TITLE
New issues about the project

### ISSUE THREAD

**Issue by BlindSable8** [BlindSable8/Final#3]
There are some errors when we try to connect the subpage to our main website, we need to double-check our code.

**Comment by BlindSable7** [comment]
Yes, I also found that issue. And I will rewrite some code for our main page.
